edxval/api.py

```diff
@@ -277,17 +278,62 @@ def get_videos_for_ids(
     return (VideoSerializer(video).data for video in videos)
 
 
-def get_video_info_for_course_and_profile(course_id, profile_name):
+def get_video_info_for_course_and_profiles(course_id, profiles):
     """
-    Returns a dict mapping profiles to URLs.
+    Returns a dict of edx_video_ids with a dict of profiles.
 
     If the profiles or video is not found, urls will be blank.
```

Comment:
This seems to be out of date. I think the description should explicitly state that a video without an encoding in one of the specified profiles will not be in the result.